cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/server.go

```diff
@@ -121,19 +128,8 @@ func (s *Server) GetAvailablePackageSummaries(ctx context.Context, request *core
 		return nil, status.Errorf(codes.Unimplemented, "Not supported yet: not including a namespace means that it returns everything a user can read")
 	}
 	// After requesting a specific namespace, we have to ensure the user can actually access to it
-	// If checking the global namespace, allow access always
-	hasAccess := namespace == s.globalPackagingNamespace
-	if !hasAccess {
-		var err error
-		// If checking another namespace, check if the user has access (ie, "get secrets in this ns")
-		hasAccess, err = s.hasAccessToNamespace(ctx, namespace)
-		if err != nil {
-			return nil, status.Errorf(codes.Internal, "Unable to check if the user has access to the namespace: %s", err)
-		}
-		if !hasAccess {
-			// If the user has not access, return a unauthenticated response, otherwise, continue
-			return nil, status.Errorf(codes.Unauthenticated, "The current user has no access to the namespace %q", namespace)
-		}
```

Comment:
This was already present in get summaries and get details, and I was going to need the same functionality, so pulled it out to a separate function.

This also means that we don't need to test this exhaustively in the actual unit tests for each handler (but should test it separately).